packages/bubble-dev/start-preset/src/pkg.ts

```diff
@@ -62,44 +62,51 @@ export const Pkg = (replacers?: TReplacers) => (packagePath: string, shouldAddMi
     }
 
     let type = null
+    let name = null
 
-    if (templateNames.length === 1) {
-      type = templateNames[0]
+    if (Boolean(templateName) && templateNames.includes(templateName)) {
+      type = templateName
     } else {
-      const result = await prompts(
+      if (templateNames.length === 1) {
```

Comment:
can we add some comet here? something for future us to know in which stage of the script are we?

Reply:
not sure I understand, wdym?

Reply:
i had in mind something to explain what does it mean that `templateNames.length === 1` just because it always feels uncomfortable for me to see something like `type = templateNames[0]`
or you do not feel there is a reason for it?

Reply:
oh I understand now! I strive for the code to be readable on its own when possible so I made it into a function
is it clearer? https://github.com/bubble-dev/_/pull/463/commits/3714f2d34682e0e5dd5dd228cff28f07fa52bbca#diff-1c161cfb97b16f26166bb64f3112736cba3eee7b1682442e0348df90166f8cadL70